### PR TITLE
Fixed Qt selection in ROI tab, and removed repeatedly showing success/warning/error messages

### LINKED ISSUE #17942: Imaging GUI: Qt fix Select option inaccuracy depending on how the widget is placed
If there is some space around the image, i.e. the imaging window is too wide, the Select placement will be very inaccurate, depending on how much space there is between the image image the bottom on the image screen, this is an issue with how the mouse position is processed in the Qt widget.

Also fixes https://github.com/mantidproject/mantid/issues/18082

### LINKED ISSUE #18082: Imaging GUI: Repeated connect leads to incremental number of warnings shown
From #17871
- If I put a "corrupt" .fits file into a folder (just a file with text mashed in on the keyboard) and attempt to load a Stack under ROI I get the same error message 4 times. However the rest of the stack loads correctly which is confusing as an error implies this failed.

and

- When I aggregate bands with `data_stack_larmor_summed` and uniform number of bands set to 143 I get a success message twice. If I set the indices of boundaries to say `12` I get the same warning 3 times.

have the same underlying problem:
https://github.com/mantidproject/mantid/blob/master/MantidQt/CustomInterfaces/src/Tomography/ImageROIPresenter.cpp#L236
and 
https://github.com/mantidproject/mantid/blob/master/MantidQt/CustomInterfaces/src/Tomography/EnergyBandsViewQtGUI.cpp#L135

connect the algorithm runner to the same slot, thus the number of warnings is increasing by 1 every time you try to load a stack of images or aggregate

### PR DESCRIPTION
**Pull request by dtasev**
Description of work.

- A little bit of refactoring in the ROI presenter
- Repeatedly loading a corrupt image should always show _only_ 1 warning/error message
- Aggregating images should now show only 1 success/error messages
- Select Centre of Rotation (COR)/Area for Normalisation/Region of Interest (ROI) should now *always* be accurate independent of the size of the window or the image position

**To test:**

- Try repeatedly loading corrupt images (could just be a file with nonsense information and extension `.fits`)
- Try aggregating a few times
- Load an image and select the COR, it should always be placed on the Mouse position
- Load an image and select the area for normalisation, ROI, the areas should always be accurate to the mouse position.

Fixes #17942 
Fixes #18082

Release Notes
----------------
[In `imaging.rst`](https://github.com/mantidproject/mantid/pull/18113/files#diff-e678fb82fbdc92d52b938c69b0858a2e)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v3.9.0/imaging.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+   :local:
+
+Tomographic reconstruction graphical user interface
+###################################################
+
+- Running local reconstructions is now possible
+
+Bug Fixes
+---------
+- The external interpreter and scripts paths are no longer ignored and are now correctly appended when running a local reconstruction
```
Comment (raquelalvarezbanos) [inline docs/source/release/v3.9.0/imaging.rst]: This fix and the above functionality ("Running local reconstruction is now possible"), are they part of this work or do they come from a different PR?

Comment (DimitarTasev) [inline docs/source/release/v3.9.0/imaging.rst]: I think that's appeared because I merged the file with #17940, it's not part of this PR

Comment (DimitarTasev) [inline docs/source/release/v3.9.0/imaging.rst]: It's been removed from the file!

**Comment by raquelalvarezbanos** [comment]
Works as described:
- Loading corrupt images prints only one error message
- COR, ROI and area for normalization they all look accurate to the mouse position.
- Code changes look good.

:shipit:
